Add email for team assumption survey (#181)
srobo/team-emails: SR2024/2024-03-12-assumption-survey.md

Author: I think that's very different. This form is about surveying the team landscape. Improvements for say onboarding rookie teams is something better suited to the competition feedback form.

---
to: SR2022 teams, SR2023 signups, SR2024 signups
subject: Help us understand what's involved in taking part
---

Hi all,

> We’re gathering some information teams, to understand the positions and opinions teams are in. If you could spare a couple of minutes to fill in this short survey, it would be much appreciated.

Author: It's the wording we used last time, I'm open to dropping it:

```suggestion
We’re gathering some information from teams, to understand the positions they are in. If you could spare a couple of minutes to fill in this short survey, it would be much appreciated.
```

> We’re gathering some information teams, to understand the positions and opinions teams are in. If you could spare a couple of minutes to fill in this short survey, it would be much appreciated.

Reviewer: Yeah, that suggested wording reads better. I'm just not sure what "opinion" this was previously referring to.
